### ISSUE TITLE
QGIS install

### ISSUE THREAD

**Issue by erichahn** [laspy/laspy#81]
I've been unable to find any decent documentation on how to install this for use with QGIS on windows. Where should this be installed and how?

**Comment by grantbrown** [comment]
I don't know whether anyone on the team uses QGIS or not, and I haven't used it for any serious projects. In any case, this question is outside of our scope, but I've attempted to put together a brief, Googled answer anyway. 

Laspy is a python library, so it is installed in the usual way described in the documentation. It does look like QGIS provides a Python console and a package exposing its API, but it also [looks like this is not dependent on the system Python install](https://www.lutraconsulting.co.uk/blog/2016/03/02/installing-third-party-python-modules-in-qgis-windows/).  That link may be helpful in getting the package installed. 

I will also say that we certainly do not have any built-in support for the QGIS API, so once you get it installed it's up to you to figure out how to get QGIS to work with the data. [This link](https://gis.stackexchange.com/questions/19338/viewing-lidar-data-las-in-qgis) may be helpful in this regard, or in suggesting alternate tools.

**Comment by erichahn** [comment]
I was able to find another way to do this as well. I had seen may references to OSGeo4W Shell. I finally found this shell in Program Files/QGIS 2.18/OSGEO4W.bat. Open this with a command prompt, or the shortcut installed in a folder on the desktop. Once inside here all pip commands *should* work. At first I had an issue with using anything pip related.

python -m pip install --upgrade pip
python -m pip install setuptools
python -m pip install laspy

and that seemed to get things working.
